spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'FoodFacade' do 
+
```

Comment:
Nice job testing for both keyword and count. I wonder how we might be able to just make one call here to get both keyword and count?
```
it 'returns an array of searched food object info', :vcr do
    search_results = FoodFacade.foods_by_keyword("sweet potatoe")
    expect(search_results[1]).to be_a Array
    expect(search_results[1].count).to eq(10)

    food_data = search_results[1].first
    expect(food_data.upc).to be_a(String)
    expect(food_data.description).to be_a(String)
    expect(food_data.company).to be_a(String)
    expect(food_data.ingredients).to be_a(String)

    total_hits = search_results[0]
    expect(total_hits).to be_a(Integer)
  end
```
Also, we will want to use vcr or webmock to stub our calls.